src/components/MailboxInlinePicker.vue

```diff
@@ -92,6 +92,7 @@ export default {
 input.vue-treeselect__input {
 	margin: 0;
 	padding: 0;
+	border: 1px solid var(--color-border-maxcontrast) !important;
```

Comment:
What makes this necessary?

Reply:
treeselect borders are 2px by default. Ours are 1px.